onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selectors.h

```diff
@@ -269,14 +273,14 @@ class DropDQNodesSelector : public BaseSelector {
 
 class UnarySelector : public BaseSelector {
  public:
-  explicit UnarySelector(bool allow_16bit = false)
-      : BaseSelector(std::make_unique<UnaryNodeGroupSelector>(allow_16bit)) {}
+  explicit UnarySelector(std::vector<const char*> compatible_providers, bool allow_16bit = false)
```

Comment:
Same thing here, using a span would be the preferred semantics.

Reply:
Also, is there a reason this one doesn't take a default `{}` parameter but all the other selectors do?